extension/observer/k8sobserver/factory_test.go

```diff
@@ -24,26 +24,29 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.uber.org/zap"
-	"k8s.io/client-go/rest"
+	"k8s.io/client-go/kubernetes"
+
+	k8sconfig "github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/kubernetes"
```

Comment:
This proves that a better name is `k8sconfig` for the package.

Reply:
Done.